corehq/apps/reports/static/reports/js/filters/button_group.js

```diff
@@ -20,16 +20,6 @@ hqDefine("reports/js/filters/button_group", ['jquery'], function ($) {
             }
             $activeCheckbox.trigger('change');
 
-            if ((!$el.children().hasClass('btn-primary')) && !canBeEmpty) {
```

Comment:
`canBeEmpty` can be removed from the param list - I'm surprised the linter didn't flag this.

Reply:
https://github.com/dimagi/commcare-hq/pull/23238/commits/016178d497704962133d6e78f6b90130bd3f6d78